src/fileTransformers/engineType.ts

```diff
@@ -0,0 +1,26 @@
+import { FileTransformer } from '../fileTransformer/fileTransformer'
+
+export const engineType: FileTransformer = (params) => {
+  const { file, parameters } = params
+
+  let content = file.content
+
+  if (file.path === 'prisma/schema.prisma') {
+    if (parameters.engineType) {
+      content = content.replace(
+        `provider = "prisma-client-js"`,
+        `provider = "prisma-client-js" \n engineType = "${parameters.engineType}"`
+      )
+    }
+  }
+  if (file.path === 'package.json') {
+    if (parameters.engineType) {
```

Comment:
Too late now, but @eemmiillyy make sure you fix this later. Just the erxistence of `engineType` is not enough to replace the version. It has to also be equal to `dataproxy`.